stylesheets/components/badge.css

```diff
@@ -0,0 +1,47 @@
+@import url(/stylesheets/main.css);
+
+.badge-container {
+  width: fit-content;
+  position: relative;
+  margin: 1rem;
+}
+
+.bdg-i {
+  font-size: 2rem;
+}
+
+.badge-icon {
+  left: 60%;
```

Comment:
where did you use this class? Didn't get this.

Reply:
I have used this class to position badge for my icons.